posthog/migrations/0212_deprecated_old_tags.py

```diff
@@ -0,0 +1,134 @@
+# Generated by Django 3.2.5 on 2022-02-12 00:17
+
+import django.contrib.postgres.fields
+import django.db.models.deletion
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("posthog", "0211_async_migrations_errors_length"),
+    ]
+
+    operations = [
+        migrations.RenameField(model_name="dashboard", old_name="tags", new_name="deprecated_tags",),
+        migrations.RenameField(model_name="insight", old_name="tags", new_name="deprecated_tags",),
+        migrations.AlterField(
+            model_name="dashboard",
+            name="deprecated_tags",
+            field=django.contrib.postgres.fields.ArrayField(
+                base_field=models.CharField(max_length=32),
+                blank=True,
+                db_column="tags",
```

Comment:
I think that makes sense. For renaming columns I don't think anything could go wrong with renaming the column since the codebase is a monolith and we're changing all references to "tags" anyways